modules/contact/test/tests/bouncing-block-contact/tests

```diff
@@ -184,4 +184,8 @@
       allow_test_objects = True
     []
   []
+  [bouncing-block-ranfs-action]
+    type = 'RunApp'
+    input = 'bouncing-block-ranfs-action.i'
```

Comment:
Not sure why this didn't fail. I'll have to have @aeslaughter look at this. Please add a requirement to this test specification. You don't need to add "issues" or "design" because they are in the top block. However, I would still encourage you to add to the existing "issues" parameter because this RANFS formulation is new and it would be nice to connect it to this change.

Reply:
Thanks for checking this @permcody ! This new test would be part of the RANFS development (i.e., existing issue 13080). So I did not add separate issue number. A requirement description has been added. Let me know it there would be further issues @permcody @aeslaughter

Reply:
Okay, I find that issue #14454 should be a better reference here. I added the issue number to the test in an updated branch in the new PR #14699